### DISCUSSION TITLE
Preserve special chars like # or @ for searching

### DISCUSSION THREAD

**Discussion by DavertMik** [meilisearch/product#480] [Feedback & Feature Proposal]
If we use hashtags or user names inside documents (think of tweets), we need to ensure that `#` or `@` char is taken into account.
However, Meilisearch treats `#` or `@` as a blank character. So `#special` is the same as `special`. 
Can we configure the list of empty characters for indexing, so `@` or `#` won't be ignored? 
This applies very well to the idea of prefix-based search, so I hope it is possible to implement.

**Comment by gmourier** [comment]
Hi @DavertMik 👋

Indeed, this is not possible yet. This is related to the tokenizer separators. We have a more detailed discussion [here](https://github.com/meilisearch/product/discussions/160) for this subject.

I'm locking this discussion to favor #160.

Thank you!
